index.html

```diff
@@ -109,6 +109,9 @@ <h6 class="dropdown-header">RS3 Discords</h6>
                 <li class="nav-item">
                     <a class="nav-link expanding_button" href="#" id="layout-button" title="Toggle Layout">⊟ <span class="expanding_text"> Compact Mode</span></a>
                 </li>
+                <li class="nav-item">
+                    <a class="nav-link expanding_button" href="#" id="token-button" title="Import / Export" data-bs-toggle="modal" data-bs-target="#token-modal">🖫 <span class="expanding_text">&nbsp;Import / Export</span></a>
```

Comment:
```suggestion
                    <a class="nav-link expanding_button" href="#" id="token-button" title="Import / Export" data-bs-toggle="modal" data-bs-target="#token-modal">🖪 <span class="expanding_text">&nbsp;Import / Export</span></a>
```
Does this icon work on your system? The other one isn't working for me